freqtrade/exchange/__init__.py

```diff
@@ -323,7 +340,12 @@ def sell(self, pair: str, ordertype: str, amount: float, rate: float) -> Dict:
             amount = self.symbol_amount_prec(pair, amount)
             rate = self.symbol_price_prec(pair, rate) if ordertype != 'market' else None
 
-            return self._api.create_order(pair, ordertype, 'sell', amount, rate)
+            if time_in_force == 'gtc':
+                return self._api.create_order(pair, ordertype, 'sell', amount, rate)
+            else:
+                return self._api.create_order(pair, ordertype, 'sell',
+                                              amount, rate, {'timeInForce': time_in_force})
```

Comment:
i would like to see a test for this aswell - checking if it's passed to ccxt as expected.
Not because i don't trust it's happening (i see that in the code) - but to avoid regressions should we refactor some of this ...